src/Http/Http.Extensions/test/HttpResponseJsonExtensionsTests.cs

```diff
@@ -302,6 +303,7 @@ async IAsyncEnumerable<int> AsyncEnumerable()
     }
 
     [Fact]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/45557")]
```

Comment:
This bug was fixed in runtime and the tests are no longer failing in the quarantine run. Let's revert the quarantine.